components/slider/index.js

```diff
@@ -68,8 +91,33 @@ export class Slider extends PureComponent {
     );
   };
 
+  handleOnChange = (newValue) => {
+    const { value, disableStartHandle, disableEndHandle, onChange } = this.props;
```

Comment:
@willian-viana `value` is a prop name defined by the third party library we're using. See the docs [here](https://github.com/schrodinger/rc-slider) (note that we're using an older version). It's the one prop/variable I did not name. Since we can't change the third party library, `newValue` seemed consistent for use in this one function where we'll be manipulating `value`. 

I do agree it's a bit confusing that it can be either a number or an array of numbers, but it's how the third party library set it up.

I had a look on whether I could change the naming and/or calculations in other places that make use of the `Slider` component, but it'd be a lot of work and bug prone because the calculations on these values are not straightforward to understand, and it'd force us to refactor not only the calculations, but the components that use this as well as track down all the widgets that use this and refactor them if needed. 

All of which seems like it's out of scope for this task, and a lot of work just to change one variable name.

Reply:
Hi, y'all! This PR comment from @willian-viana just caught my eye this morning. I understand what you're saying @SARodrigues, and I appreciate having a codebase that better communicates its intent (what @willian-viana is suggesting). Couldn't we alias the prop here where it's being destructured so that the function's intent is more precise given that `value` is so generic and ambiguous?
For example:
```javascript
const { value: sliderPositions, disableStartHandle, disableEndHandle, onChange } = this.props;
```
Or whatever better naming is appropriate. 
Just a thought. 🤷

Reply:
Tweaks made using @gtempus suggestion!

Reply:
Thank you for your suggestions, @gtempus! Thank your for the change @SARodrigues (: